database/seeds/eggs/voice-servers/egg-teamspeak3-server.json

```diff
@@ -3,34 +3,34 @@
r\ndo\r\n    # If no log file is there, we can't retrieve the last installed version and have to install the latest\r\n\tif [ ! -f \"$FILE\" ]; then\r\n\t\tVERSION=$(curl -s 'https:\/\/www.teamspeak.com\/versions\/server.json' | jq -r '.linux.x86_64.version')\r\n\t\tLATESTTS3RELEASELINK=$(curl -v 'https:\/\/www.teamspeak.com\/versions\/server.json' | jq -r '.linux.x86_64.mirrors.\"teamspeak.com\"')\r\n\t\techo -e \"\\nDownloading TS3 Version $VERSION\\n\"\r\n\t\tcurl -v $LATESTTS3RELEASELINK | tar xj --strip-components=1\r\n\telse\r\n\t    # Install latest if update is set to true\r\n\t\tif [ \"${UPDATE}\" -eq 1 ]; then\r\n\t\t\tVERSION=$(curl -s 'https:\/\/www.teamspeak.com\/versions\/server.json' | jq -r '.linux.x86_64.version')\r\n\t\t\tLATESTTS3RELEASELINK=$(curl -s 'https:\/\/www.teamspeak.com\/versions\/server.json' | jq -r '.linux.x86_64.mirrors.\"teamspeak.com\"')\r\n\t\t\techo -e \"\\nDownloading TS3 Version $VERSION\\n\"\r\n\t\t\tcurl $LATESTTS3RELEASELINK | tar xj --strip-components=1\r\n\t\t# Otherwise get the log file and grep the version from a certain place. Then get latest offical download link and replace the version with the old one. With that we have a dynamic downloadlink since teamspeak hosts all versions in the same structur\r\n\t\telse\r\n\t\t\tINSTALLEDVERSION=$(cat $(find .\/* -name \"ts3server*_0.log\" 2> \/dev\/null | sort | grep -Ev \"${rootdir}\/.ts3version\" | tail -1) | grep -Eo \"TeamSpeak 3 Server ((\\.)?[0-9]{1,3}){1,3}\\.[0-9]{1,3}\" | grep -Eo \"((\\.)?[0-9]{1,3}){1,3}\\.[0-9]{1,3}\" | sort -V | tail -1)\t\r\n\t\t\tOLDTS3RELEASELINK=$(curl -s 'https:\/\/www.teamspeak.com\/versions\/server.json' | jq -r '.linux.x86_64.mirrors.\"teamspeak.com\"' | sed -r \"s\/((\\.)?[0-9]{1,3}){1,3}\\.[0-9]{1,3}+\/$INSTALLEDVERSION\/g\")\r\n\t\t\techo -e \"\\nDownloading TS3 Version $INSTALLEDVERSION\\n\"\r\n\t\t\tcurl $OLDTS3RELEASELINK | tar xj --strip-components=1\r\n\t\tfi\r\n\tfi\r\n\t# Just do it one time\r\n\tbreak\r\ndone\r\n\r\n# Accept license\r\nif [ ! -f \".ts3server_license_accepted\" ]; then\r\n\ttouch \".ts3server_license_accepted\"\r\nfi\r\n\r\n# Add default config if not there\r\nif [ ! -f \"ts3server.ini\" ]; then\r\n    echo \"Creating default config for teamspeak\"\r\n    echo 'machine_id=\r\ndefault_voice_port=9987\r\nvoice_ip=\r\nlicensepath=\r\nfiletransfer_port=30033\r\nfiletransfer_ip=\r\nquery_port=10011\r\nquery_ip=0.0.0.0, ::\r\nquery_ip_whitelist=query_ip_whitelist.txt\r\nquery_ip_blacklist=query_ip_blacklist.txt\r\ndbplugin=ts3db_sqlite3\r\ndbpluginparameter=\r\ndbsqlpath=sql\/\r\ndbsqlcreatepath=create_sqlite\/\r\ndbconnections=10\r\nlogpath=logs\r\nlogquerycommands=0\r\ndbclientkeepdays=30\r\nlogappend=0\r\nquery_skipbruteforcecheck=0\r\nquery_buffer_mb=20\r\nhttp_proxy=\r\nlicense_accepted=1\r\nserverquerydocs_path=serverquerydocs\/\r\nquery_ssh_ip=0.0.0.0, ::\r\nquery_ssh_port=10022\r\nquery_protocols=raw,ssh\r\nquery_ssh_rsa_host_key=ssh_host_rsa_key\r\nquery_timeout=300' > ts3server.ini\r\nfi",
```

Comment:
You default to accepting a license for a user. That would make anyone here liable for their user who never had to accept the license.

Reply:
Before the license was accepted via command line:
https://github.com/pterodactyl/panel/blob/develop/database/seeds/eggs/voice-servers/egg-teamspeak3-server.json#L11